lib/api/kuzzle.js

```diff
@@ -194,7 +213,7 @@ function registerErrorHandlers(kuzzle) {
 
   // Remove external listeners (PM2) to avoid other listeners to exit current process
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 85 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)